src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/EmptyBlockCheck.java

```diff
@@ -260,8 +260,7 @@ public void visitToken(DetailAST ast) {
                 }
                 if (emptyBlock) {
                     log(leftCurly,
-                        MSG_KEY_BLOCK_NO_STATEMENT,
-                        ast.getText());
+                        MSG_KEY_BLOCK_NO_STATEMENT);
```

Comment:
https://github.com/checkstyle/checkstyle/search?q=block.noStatement

This message has no parameters, so the text given to it is not necessary.

Reply:
Good find, looks like there might be others like this:

```
➜  checkstyle git:(move-travis-tasks) ✗ ag -Q "removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText"
config/pitest-suppressions/pitest-coding-1-suppressions.xml
89:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>
143:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>

config/pitest-suppressions/pitest-javadoc-suppressions.xml
323:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>

config/pitest-suppressions/pitest-blocks-suppressions.xml
8:    <description>removed call to com/puppycrawl/tools/checkstyle/api/DetailAST::getText</description>


```

Reply:
I am going group by group for now. I am skipping over common areas and more focusing on checks.